_pages/en/strategy-version-control-systems.md

```diff
@@ -0,0 +1,346 @@
+    <ul>
+      <li>Continuous Integration (CI)</li>
+      <li>Continuous Deployment (CD)</li>
+    </ul>
+    </td>
+    <td><b>SSC</b></td>
+    <td>
+    DevCoP<br>
+    ?
+    </td>
+  </tr>
+  <tr>
+    <td>Migrate projects to GCcode</td>
+    <td>Migrate software-related projects, scripts and system/database configurations to the chosen VCS, and gradually sunset legacy version control systems
+    </td>
+    <td><b>Development teams</b></td>
+    <td>
+    DevCoP<br>
+    Operations teams<br>
+    ?<br>
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="1"><b>Communications</b></td>
+    <td>Train staff</td>
+    <td>Offer training resources to IITB staff related to DevOps and VCS</td>
+    <td><b>Learning Branch</b></td>
+    <td>
+    DevCoP<br>
+    ?
+    </td>
+  </tr>  
+</table>
+
+## Measuring the Strategy's success
+
+This Strategy's success will be measured by comparing the following metrics against the conventional IT project methodology.
+
+The metrics are defined as follows:
+
+1. *Number of ESDC projects (open vs closed) and users in each VCS*
+1. *Level of activity of ESDC projects (e.g., issues, commits, integrations) in each VCS*
+1. *User Satisfaction*
+1. *?*
+
+<table>
+<tr>
+    <th rowspan="2">Metric</th>
+    <th colspan="2">Collection Method</th>
+</tr>
+<tr>
+    <th>Target Model</th>
+    <th>Conventional Model</th>
+</tr>
+<tr>
+    <td>Number of ESDC projects (open vs closed) and users in each VCS</td>
+    <td>
+    <i>Automatic</i><br>
+    Automatically fetch metrics from platforms
+    </td>
+    <td>
+    <i>Manual</i><br>
+    Manually copy metrics from platforms
+    </td>
+</tr>
+<tr>
+    <td>Level of activity of ESDC projects (e.g., issues, commits, integrations) in each VCS</td>
+    <td>
+    <i>Automatic</i><br>
+    Automatically fetch metrics from platforms
+    </td>
+    <td>
+    <i>Manual</i><br>
+    Manually copy metrics from platforms
+    </td>
+</tr>
+<tr>
+    <td>User Satisfaction</td>
+    <td>
+    <i>Manual</i><br>
+    Client survey
+    </td>
+    <td>
+    <i>Manual</i><br>
+    Client survey
+    </td>
+</tr>
+<tr>
+    <td>?</td>
+    <td>
+    TBC
+    </td>
+    <td>
+    TBC
+    </td>
+</tr>
+</table>
+
+*Manual*: the collection of data requires manual intervention (e.g., surveys, interviews, emails, spreadsheet updates).
+
+*Automatic*: the collection of data is performed automatically, usually involving programmatic means (e.g., events triggered by a Git repository when a new commit is performed, which updates a master dashboard “view”).
+
+## Appendix A - Definitions
+
+Version Control System (VCS)
+: There are 2 types of VCS:
+
+- **Centralized** VCS are based on the idea that there is a single “central” copy of a software project somewhere (most likely on a server), and developers make code changes directly on this central copy.
+- **Decentralized** VCS (DVCS) do not necessarily rely on a central server to store all the versions of a software project's files.
```

Comment:
I'd suggest "Distributed" instead of "Decentralized", I think this is the standard term.

Reply:
Good catch, thanks! Corrected.